reinforcement_learning/rl_roboschool_ray/Dockerfile

```diff
@@ -29,7 +29,10 @@ RUN curl -fSsL -O https://bootstrap.pypa.io/get-pip.py && \
 
 RUN pip install --upgrade \
     pip \
-    setuptools
+    setuptools \
```

Comment:
Does the current Docker file not work?

Reply:
The current Docker file works, this is for better user experience. `setproctitle` and `psutil` are for Ray dashboard and memory tracking; lz4 is sample compression which impacts RLlib's performance.